### DISCUSSION TITLE
Exclude images from sitemap

### DISCUSSION THREAD

**Discussion by nachols889** [PrestaShop/PrestaShop#35242] [Ideas]
### The problem

Hello. I am using "Google sitemap" and just realized that the generated sitemap includes images, which makes sitemap very long. 

### The solution

I would need to exclude images from prestashop so only category URLs are shown in sitemap. 

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
Hello @nachols889 

It could be included as an option for the https://github.com/PrestaShop/gsitemap module, but I don't think project members will have time to implement it soon. If you, or any community member, are motivated to submit a pull request implementing this feature, we will be happy to help getting it merged :)
